components/eamxx/src/control/intensive_observation_period.cpp

```diff
@@ -725,18 +725,14 @@ read_iop_file_data (const util::TimeStamp& current_ts)
       // the interpolated region with the value at model_start/model_end
       if (fname == "T"    || fname == "q" || fname == "u" ||
           fname == "u_ls" || fname == "v" || fname == "v_ls") {
-        if (model_start > 0) {
-          Kokkos::parallel_for(Kokkos::RangePolicy<>(0, model_start),
-                               KOKKOS_LAMBDA (const int ilev) {
-            iop_field_v(ilev) = iop_field_v(model_start);
-          });
-        }
-        if (model_end < total_nlevs) {
-          Kokkos::parallel_for(Kokkos::RangePolicy<>(model_end, total_nlevs),
-                               KOKKOS_LAMBDA (const int ilev) {
-            iop_field_v(ilev) = iop_field_v(model_end-1);
-          });
-        }
```

Comment:
@bogensch Previously, EAMxx was doing 2 things different to EAM for this subset of iop fields (in EAM, fields that have `fill_ends=true`)
i. EAMxx was setting the non-interpolated region to the first/last interpolated value in the IOP field, whereas in EAM we set it to the first/last value in the IOP file.
ii. EAM always overwrites the first and last value in the interpolated region to the IOP file start and end values, whereas EAMxx was only setting end points outside the interpolated region.

Let me know if this is the intended behavior. I've verified the IOP fields now match exactly between EAM and EAMxx.